script/releases/EigenLabsUpgrade.s.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
```

Comment:
@nadir-akhtar @wadealexc  this previously lived in the zeus-templates repo which made no sense and bled information about the state of the EL protocol into the core of zeus... moved this here.